### PR TITLE
Added digital ocean example 

### PR DESCRIPTION
**Pull request by MyDigitalLife**
As asked in #146 i created a example for digital ocean. I have not been able to test this because there isn't really any documentation on how to use this.

### CODE REVIEW

**Comment by looztra** [comment]
the digitalocean profile and code is already available, see https://www.nivenly.com/kubernetes-on-digital-ocean-with-encrypted-vpn-service-mesh/

**Comment by MyDigitalLife** [comment]
Hmm the Travis build is failing because there are now 2 files with the main function in the same directory.  I can edit the Makefile to exclude the examples but that would mean that the files are never tested so it might not be the best solution. I think it might be a better idea to sub folder the examples (if that fixes it). So it would be something like this:
```
examples/digitalocean/example1.go
examples/aws/example2.go
```

What do you think @kris-nova

**Comment by kris-nova** [comment]
I vote nested folders `examples/$cloud/$example.go`

File: examples/digitalocean/basiccluster.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+		panic(err.Error())
+	}
+	err = reconciler.Destroy()
+	if err != nil {
+		panic(err.Error())
+	}
+}
+
+func getCluster(name string) *cluster.Cluster {
+	return &cluster.Cluster{
```
Comment (kris-nova) [inline examples/digitalocean/basiccluster.go]: Can we make the profile match this one please? https://github.com/kris-nova/kubicorn/blob/master/profiles/simpledigitalocean.go#L23

**Pull request merged**